### ISSUE TITLE
Add settings item - Remember the position of the add download window

### ISSUE THREAD

**Issue by wvxwxvw** [jingyu9575/multithreaded-download-manager#107]
Add settings item - Remember the position of the add download window
By analogy with  - Remember the position of the downloads window.

**Comment by jingyu9575** [comment]
That option should work for both the downloads window and the *add download* window

**Comment by wvxwxvw** [comment]
However, this is not the case. Absolutely.

**Comment by wvxwxvw** [comment]
Yes, this I have that the prevents. Is it possible to fix it somehow? By what landmarks is the position of the window remembered, by the main Firefox window or by the corners of the screen?

I have the title bar hidden and the tab bar.

**Comment by jingyu9575** [comment]
Looks like the problem is also *Open extension windows in Private Browsing mode*. When the window is private, the extension is unable to save data on closing. 

I think this option is causing many troubles. The original purpose is to prevent the extension windows from showing in the history and the undo closing list (#53 in Chinese). An alternative way is erasing the history items when the window is closed, but this requires *history* and *sessions* permission, and I think many users do not want more permissions now.
